projects/frontend-ui/shared-components/projects/gui/README.md

```diff
@@ -0,0 +1,69 @@
+The Shared Components contains:
+
+* **[Shared Components Library](projects/shared/README.md)** Library that holds UI Components.
+* **[Documentation UI](projects/documentation-ui/src)** Reference UI Wrapper and Documentation Implementation
+
+
+## Running and Testing
+
+You can use the reference implementation in `projects/documentation-ui` as showcase of the library.
+Using `npm link` you can achieve real-time development of the library without the need to upload it to repository
+
+_**Note**_: If you run directly npm install without linking it will tell you that there is no such package.
+That's why you need to link it the first time you build your development environment.
+
+Refer to [.gitlab-ci.yml](../../cicd/.gitlab-ci.yml) file for correct setup guaranteed to work by automation.
+Bug in general the steps are:
+
+0. Set npm registry to use
+   * There is .npmrc file in the project that refers to correct registries and provide configuration
+   * If something is not working with autoconfiguration, manually set registries
+
+   ```bash
+   $ npm config set legacy-peer-deps true
+   ```
+1. Install Angular Cli and all other dependencies except '@taurus/shared'
+   ```bash
+   npm i -g @angular/cli
```

Comment:
isn't there a specific version compatibility? we should add this, and also describe it in the VEP

Reply:
@ivakoleva  why do you think there is a specific version compatibly? 
I just run this command as described and everything worked correctly.

Reply:
I think that we shouldn't specify the version as they will have other projects using this package too.
Given that we are expecting them to use the global angular/cli package we should work with as many versions as possible.
We would want to know asap if it doesn't work with a newer version or any version ever.